app/views/pay/user_mailer/receipt.html.erb

```diff
@@ -7,7 +7,7 @@ RECEIPT - SUBSCRIPTION<br/>
 <br/>
 Amount: USD <%= ActionController::Base.helpers.number_to_currency(@charge.amount / 100.0) %><br/>
 <br/>
-Charged to: <%= @charge.card_brand %> (**** **** **** <%= @charge.card_last4 %>)<br/>
+Charged to: <%= @charge.card_type %> (**** **** **** <%= @charge.card_last4 %>)<br/>
```

Comment:
This reminded me, I want to go through and standardize everything to either `card_brand` or `card_type`. I believe Stripe uses `brand` and Braintree uses `type`. Right now we're generally processing credit cards + PayPal, but Braintree supports Venmo and some other options. I'm not sure the best wording for this, but I'll open up another issue for it.

Reply:
For now I think we'd be fine to merge this, and then fix it in another commit.